packages/auth-remix/src/server.ts

```diff
@@ -313,6 +318,13 @@ export class RemixServerAuth extends RemixClientAuth {
                 path: "/",
               }),
             );
+            headers.append(
```

Comment:
Why are we deleting this verifier cookie here and in the other two places?

Reply:
u are right, this is not needed... we delete the pkce when/after verifying

Reply:
hmm let me check what happens if user started signing up flow but didnt finish, while in the meantime the lib version changed